src/main/java/net/mcreator/ui/modgui/ToolGUI.java

```diff
@@ -269,12 +270,12 @@ public ToolGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 
 		page1group.addValidationElement(texture);
 
-		name.setValidator(new TextFieldValidator(name, "Tool needs a name"));
+		name.setValidator(new TextFieldValidator(name, L10N.t("elementgui.tool.needs_a_name")));
 		name.enableRealtimeValidation();
 
-		addPage("Visual", pane2);
-		addPage("Properties", pane4);
-		addPage("Triggers", pane3);
+		addPage(L10N.t("elementgui.tool.page_visual"), pane2);
+		addPage(L10N.t("elementgui.tool.page_properties"), pane4);
+		addPage(L10N.t("elementgui.armor.triggers"), pane3);

```

Comment:
Why use "elementgui.armor.triggers" translation entry from armour when this is ToolGUI? Wouldn't it make more sense to create a common elementgui entry instead (if we want to avoid duplication) or at least make it a separate toolgui entry?

Reply:
Done. I added common keys and changed the armor ones.